src/jit-format/jit-format.cs

```diff
@@ -563,6 +568,54 @@ public static void ParseBuildLog(string buildLog, string compileCommands, string
             }
         }
 
+        public static string rewriteCompileCommands (string compileCommandFile, string project)
+        {
+            string[] compileCommands = File.ReadAllLines(compileCommandFile);
```

Comment:
I was having a hard time finding documentation to help me figure out the errors I was seeing when I tried to parse it as JSON, so I went with this.